changelog_unreleased/html/pr-7555.md

```diff
@@ -0,0 +1,17 @@
+#### Format HTML attribute `class` value ([#7555](https://github.com/prettier/prettier/pull/7555) by [@fisker](https://github.com/fisker))
+
+<!-- prettier-ignore -->
+```html
+<!-- Input -->
+<div class="  foo
+bar baz"></div>
+
+<!-- Prettier stable -->
+<div
+  class="  foo
+bar baz"
+></div>;
+
+<!-- Prettier master -->
+<div class="foo bar baz"></div>
+```
```

Comment:
Can you add example with many clases when we don't fit in line?

Reply:
done

Reply:
/cc @prettier/core What do you think, maybe we should print each class on newline if we don't fix in line for readability?

Reply:
hmm.. that's quite an edge case.. but maybe ignore the filelength in that case?

```html
<div
  class="another element with so many classes even can not fix one line really a lot and lot of classes"
></div>
```

WDYT?

Reply:
@evilebottnawi I tried print each on newline , seems not good, it's common to have many classes, and we even don't print css on newline (#7556) , css normally longer then class names.

Reply:
@lipis I don't think this is a edge case, and not a good idea to ignore the line length

[This case](https://deploy-preview-7555--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEAeAFjAtgGwHwA6UABMagEYQAmAnoScUaalQJYBuTxYOAhgM78AvARC8AtOThZJOCGADWxCVJnk5i8eIDurGOnFZqrAGZwAToxBE8zNpwY8Bw0Sumz5CgPpe4OaQgwypLu6p7ifgGwWrr6hsZmlrw4-BDivFAQ+hYeiqI2LBxcToIiYiFqGt6+-liBwaq5ChG1gTF6BkZsicopaRlZ6Dlhij6RdbD5UHioAPT29KTM8xyLZCvseCAANCAQAA4wrND8yKC85uYQ2gAKFwinKMnavDSnu+TmvIpwMADK+2+rCgAHNkDBzABXOC7TC4ADq6D0cH4gLAcD+Dz0HD0NGQ4EEOxAwP4FhgNy+IKwvGQJmSpN2ACt+AAPABCXx+-14dQAMsC4LT6TCQMyWX9gSD-ABFSFZQVIOkpEWA8yk8z4uE4In7czAmDw1hUfTIAAcAAZdrqIKT4V99vjdSiLOxBbsAI5y+AUg6PMT8cRQOBwKghonmOCe1gRim8Kk0xXC3akrCscFQkX8SUyr0KpUMkAwXjkQ3G9DIABMuwhvFYOElAGEIFhqfiUQBWImQ0kAFWLj3zIvY0IAklBQ7A-mA9YcAILjv4wGj+IXKgC+a6AA) , I think could be a real world case.

Reply:
I think respect line width in that cases is bad idea, maybe print it in one line always is better

Reply:
It is really hard to read and will create big git diff when you add a new class

Reply:
Actually I'm ok with both, so just `.split(/\s/).join(' ')` ?  So simple, :)

Reply:
Sounds good for me to inline that, will be great to hear other opinion